src/loaders/lottie/tvgLottieProperty.h

```diff
@@ -249,10 +249,18 @@ struct LottieGenericProperty : LottieProperty
         return frame->interpolate(frame + 1, frameNo);
     }
 
+    LottieGenericProperty<T>* copy()
+    {
+        //copy, used for slot reverting
+        auto ret = new LottieGenericProperty<T>(value);
+        if (frames) ret->frames = frames;
+        else ret->value = value;
+        return ret;
+    }
+
     T& operator=(const T& other)
     {
         //shallow copy, used for slot overriding
-        delete(frames);
```

Comment:
Simply removing the memory deallocation will result in memory leaks. I think you should not remove them once the review changes have been made based on the feedback.